modules/4337/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 This changelog only contains changes starting from version 0.2.0
 
-# Version 0.3.0
+# Version 0.3.1
```

Comment:
This isn't a new version (contracts are exactly the Same). For Safe 1.3.0 there also existed two deployments because of two deployer contracts, and were just tracked in the deployments repository themselves.

Reply:
Updated 👍🏾